docs/comparison/taskflow_api.md

```diff
@@ -0,0 +1,105 @@
+# TaskFlow API: Using YAML instead of Python
+
+For users that employ lots of Python functions in their DAGs, [TaskFlow API](https://www.astronomer.io/docs/learn/airflow-decorators/) represent a simpler way to transform functions into tasks, with a more intuitive way of passing data between them.
+They were  introduced in Airflow 2 as an alternative to Airflow [traditional operators](traditional_operators.md).
+
+Below, we illustrate how to represent an Airflow DAG using TaskFlow API and how to define the same DAG using
+DAG Factory. Ultimately, both implementations use the same Airflow operators. The main difference is the language used
+to declare the workflow: while one uses Python, the other uses YAML.
+
+
+## Goal
+
+Let's say we'd like to create a workflow that performs the following:
+
+1. Create a list of [PyPI](https://pypi.org/) projects to be analysed
+2. Fetch the [statistics](https://pypistats.org/) for each of these projects
+3. Summarize the selected statistics as Markdown, using Python.
+
+We will implement all these steps using the Airflow `task` decorator, and the last task will generate a Markdown table similar to:
+
+```
+| package_name      |   last_day |   last_month |   last_week |
+|:------------------|-----------:|-------------:|------------:|
+| apache-airflow    |     852242 |     28194255 |     6253861 |
+| astronomer-cosmos |     442531 |     13354870 |     3127750 |
+| dag-factory       |      10078 |       354085 |       77752 |
+```
+
+The main logic is implemented as plain Python functions in [pypi_stats.py](../../dev/dags/pypi_stats.py):
```

Comment:
When building docs locally, it gives a warning for the link like below:
```
WARNING -  Doc file 'comparison/taskflow_api.md' contains a link '../../dev/dags/pypi_stats.py', but the target '../dev/dags/pypi_stats.py' is not found among documentation files.
```
I am wondering if we could use absolute URLs for these links to get away from those Warning messages?

Reply:
Yes, it makes sense; thanks for doing this in the other docs: #331!
Just fixed in 516c922077424d7eebb155f39edf0bb1f2d532d3